spec/rails_helper.rb

```diff
@@ -28,7 +28,9 @@
 
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.fixture_paths = [
+    Rails.root.join('spec/fixtures')
+  ]
```

Comment:
I added this to fix a deprecation warning that was appearing after the tests ran.